### PR TITLE
Arm64 fix

### PR DESCRIPTION
**Pull request by jwillmartin**
# PR Details
## Description
Added some changes to allow for Docker to be installed on different Linux distros for arm64.

## Motivation and Context
I wanted to be able to install Docker on both Ubuntu and Debian for arm64 PCs. 

## How Has This Been Tested?
Tested on both Ubuntu and Debian for correct script results. 

## Types of changes

- [ ] Defect fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that cause existing functionality to change)

## Checklist:

- [ ] I have added any new packages to the sonar-scanner.properties file
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
[V2XHUB Contributing Guide](https://github.com/usdot-fhwa-OPS/V2X-Hub/blob/develop/Contributing.md) 
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: configuration/arm64/mysql/add_v2xhub_user.bash [hunk 1]
```diff
@@ -3,7 +3,12 @@
 #set -e
 
 # Ensure mysql-client is installed
-REQUIRED_PKG="mysql-client"
+arch=$(dpkg --print-architecture)
+if [ $arch = "amd64" ]; then
```
Comment (rob-baily) [inline configuration/arm64/mysql/add_v2xhub_user.bash:7]: Why are we checking for amd64 architecture for the arm64 configuration?  I would expect that this configuration is only used on arm architectures.  Maybe the check was supposed to be arm64?

**Comment by rob-baily** [comment]
@jwillmartin Is there any background or other information on this?  I reviewed but for some stuff I don't have any context to know  what the change is for.

File: configuration/arm64/mysql/add_v2xhub_user.bash (hunk 1 above)
Comment (jwillmartin) [reply configuration/arm64/mysql/add_v2xhub_user.bash:7]: Ah, my bad. The idea at the moment was to merge both configurations into one folder, but I couldn't figure out how to get different docker images pulled from one docker-compose.yml file. Probably better to keep them as is.

**Comment by jwillmartin** [comment]
@rob-baily We used to mostly use some single-board computer that ran a version of Ubuntu. That's why this configuration file was made to pull the Ubuntu version for Docker. But now we're using Raspberry Pi's running Debian, so we need to be able to install the correct Docker version.

**Review by rob-baily — APPROVED** [review]

**Pull request merged**